packages/components/src/ListView/Items/Items.component.js

```diff
@@ -13,6 +13,27 @@ const listClasses = classNames(theme['tc-list-items'], 'tc-list-items');
 const itemsClasses = classNames(theme['tc-listview-items'], 'tc-listview-items');
 const itemContainer = classNames(theme['tc-item-container'], 'tc-item-container');
 
+/**
+ * Converts a string px size in a JS valid number
+ * @param {String} sizeInPixels
+ * @returns {Number}
+ */
+function pxToInt(sizeInPixels) {
+	if (!sizeInPixels) {
+		return 0;
+	}
+
+	const parsed = parseFloat(sizeInPixels.replace('px', ''));
+
+	return isNaN(parsed) ? 0 : parsed;
+}
+
+const TOGGLE_ALL_ROW_HEIGHT = 40;
+const ROW_LINE_HEIGHT = 13;
+const ROW_MARGIN = pxToInt(theme.rowMargin);
+const ROW_INNER_PADDINGS = pxToInt(theme.rowInnerPaddings);
+const ROW_HEIGHT = ROW_LINE_HEIGHT + ROW_MARGIN;
```

Comment:
Could you move all of these consts on the top of the file, with others consts ?

Reply:
Can't do because 2 of them use `pxToInt`